### PR TITLE
Updated README and FAQ

### PR DESCRIPTION
**Pull request by tabern**
#### Summary
Updating Readme and FAQ

#### Implementation details
n/a

#### Testing
n/a
- [ ] cluster-state-service binary built locally and unit-tests pass (`cd cluster-state-service; make; cd ../`)
- [ ] cluster-state-service build in Docker succeeds (`cd cluster-state-service; make release; cd ../`)
- [ ] daemon-scheduler binary built locally and unit-tests pass (`cd daemon-scheduler; make; cd ../`)
- [ ] daemon-scheduler build in Docker succeeds (`cd daemon-scheduler; make release; cd ../`)

New tests cover the changes: 

#### Description for the changelog
Updating README and FAQ for V1 release

#### Licensing

This contribution is under the terms of the Apache 2.0 License: Yes

#### Before merging

- [ ] cluster-state-service end-to-end tests pass. Required setup details are listed [here](https://github.com/blox/blox/blob/dev/cluster-state-service/internal/Readme.md).
- [ ] daemon-scheduler integration tests pass. Required setup details are listed [here](https://github.com/blox/blox/blob/dev/daemon-scheduler/internal/features/README.md).
- [ ] daemon-scheduler end-to-end tests pass. Required setup details are listed [here](https://github.com/blox/blox/blob/dev/daemon-scheduler/internal/features/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,19 @@
...
+Blox schedulers are built using AWS primitives, and the Blox designs and code are open source. If you are interested in learning more or collaborating on the designs, please read the [design](docs/daemon_design.md)
+
+If you are currently using Blox v0.3, please read the [v0.3 FAQ](docs/faq.md)
 
 ### Contact
 
 * [Gitter](https://gitter.im/blox)
 * [Planning/Roadmap](https://github.com/blox/blox/milestones)
 * [Issues](https://github.com/blox/blox/issues)
+* [Blox Homepage](https://blox.github.io)
```
Comment (shubharao) [inline README.md:20]: Recommend adding this link to the list once the homepage is updated, so that we do not confuse new readers.

Comment (tabern) [inline README.md]: can do. Would that require opening a new pull request?

File: docs/faq.md [hunk 2]
```diff
@@ -16,11 +16,14 @@ You can use Blox’s scheduling capabilities using Amazon ECS console or using t
 * Enable customers to build and run the same schedulers that power Amazon ECS;
 * Encourage transparent product feedback and community contributions.
 
-##### Is there any cost to using Blox?
```
Comment (narehayrapetyan) [inline docs/faq.md:19]: Why are you removing the question?

File: docs/faq.md [hunk 3]
```diff
@@ -16,11 +16,14 @@ You can use Blox’s scheduling capabilities using Amazon ECS console or using t
...
-If using the managed Blox service, there is no cost as Blox is provided as part of Amazon ECS.  
-
 ##### What is the goal of Blox v1.0 compared to Blox v0.3?  
-Customers wanted to see greater convergence across the open source Blox project and Amazon ECS schedulers, and asked for a managed or hosted option for Blox. Therefore, we are providing Blox as a managed option and making it available for use with ECS without having to deploy, run and manage it.
+Customers wanted to see greater convergence across the open source Blox project and Amazon ECS schedulers, and asked for a managed or hosted option for Blox. Therefore, we are providing Blox as a managed option and making it available for use with Amazon ECS without having to manually install, deploy, and run it.
+
+##### What are we doing with cluster state service? Can I continue to use it to build custom scheduling capabilities?
+Blox provides an Event Stream Handler (cluster state service) that enables extensibility and it will continue to be available. We are looking for partner and customer feedback about their extensibility use cases as we decide on future investment into this service.
```
Comment (narehayrapetyan) [inline docs/faq.md:23]: Does saying "it will continue to be available" imply the level of support?

**Pull request closed without merging**